tools/accuracy_checker/requirements-extra.in

```diff
@@ -30,7 +30,7 @@ pyclipper>=1.2.1
 # brain tumor segmentation
 nibabel>=3.2.1
 # nncf's upperbound: https://github.com/openvinotoolkit/nncf/blob/a8af39627647ddbb89ef33cf42ff42e9c048b981/setup.py#L106
-scipy>=1.5.4
+scipy>=1.11.1
```

Comment:
How does that resolve the original problem? The build under https://github.com/openvinotoolkit/openvino/pull/19458: https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/ie/job/e2e-tests-linux-ubuntu20-pip-conflicts/40763/ fails with `ERROR: Could not find a version that satisfies the requirement pip==23.1.2`